src/storage/mvcc/reader/scanner/backward.rs

```diff
@@ -223,19 +244,58 @@ impl<S: Snapshot> BackwardKvScanner<S> {
         // At this time, we must have current commit_ts <= ts. If commit_ts == ts,
         // we don't need to seek any more and we can just utilize `last_version`.
         if last_checked_commit_ts == ts {
```

Comment:
When moving cursor backward, there could be a chance that ts of `pointing key == ts && REVERSE_SEEK_BOUND is hit`. Before this PR, it returns directly since a suitable ts is already found. In this PR however, before returning data, we need to move backward again to see if there are larger ts available to invalidate the cache.